src/views/__tests__/TransactionSigner.test.jsx

```diff
@@ -0,0 +1,259 @@
+/**
+ * @jest-environment jsdom
+ */
+import { Networks, TransactionBuilder } from "stellar-sdk";
+import { fireEvent, waitFor, screen } from "@testing-library/react";
+import * as freighterApi from "@stellar/freighter-api";
+import albedo from "@albedo-link/intent";
+import { render } from "helpers/testHelpers";
+import TransactionSigner from "views/TransactionSigner";
+import Libify from "helpers/Libify";
+import * as transactionSignerActions from "actions/transactionSigner";
+
+jest.mock("@stellar/freighter-api", () => ({
+  isConnected: () => new Promise((resolve) => resolve(true)),
+  signTransaction: () => new Promise((resolve) => resolve(SIGNED_TRANSACTION)),
+}));
+
+jest.mock("@albedo-link/intent", () => ({
+  tx: () =>
+    new Promise((resolve) =>
+      resolve({ signed_envelope_xdr: SIGNED_TRANSACTION }),
+    ),
+}));
+
+const TRANSACTION_XDR =
+  "AAAAAgAAAADYMr0vKfmdLXQTWwns9YavTU2ZCDzl7+L/dWEqn05dqQAAAGQAABjxAAAACwAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAABQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const FEE_BUMP_XDR =
+  "AAAABQAAAACafGsfMQylseeF9Du5owr05nhq/IDVl47HGz6PHGwWfwAAAAAAAADIAAAAAgAAAADYMr0vKfmdLXQTWwns9YavTU2ZCDzl7+L/dWEqn05dqQAAAGQAABjxAAAACwAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAABQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
+const TRANSACTION = {
+  network: Networks.TESTNET,
+  hash: "00f6f2a06f339bcfca7a530d10edb014983c8e752fced4d4a0d8a274c5ddb416",
+  source: "GDMDFPJPFH4Z2LLUCNNQT3HVQ2XU2TMZBA6OL37C752WCKU7JZO2S52R",
+  sequence: "27423366184971",
+};
+const SECRET_KEY = "SB3NTRERB67ZPAFLLP3G6ZFUHYKFEQXX2D7RDWSRUZ5PSHRB7QJVASUU";
```

Comment:
Secret key of a test account I made on Testnet. I don't think there are any security implications to this, but let me know!